temporalnexus/link_converter.go

```diff
@@ -62,7 +61,7 @@ var (
 	))
 )
 
-// ConvertLinkWorkflowEventToNexusLink converts a Link_WorkflowEvent type to Nexus Link.
+// EXPERIMENTAL: ConvertLinkWorkflowEventToNexusLink converts a Link_WorkflowEvent type to Nexus Link.
```

Comment:
Godoc should continue to start with the thing it's documenting, can put that it's experimental below. For example, here is what other Nexus APIs are doing in this repository:

```suggestion
// ConvertLinkWorkflowEventToNexusLink converts a Link_WorkflowEvent type to Nexus Link.
//
// NOTE: Experimental
```